front-end/my-vite-app/src/Survey.jsx

```diff
@@ -79,11 +72,9 @@ const Survey = ({username}) => {
             {step === 4 && <div className = "top-survey">
                 <h3>What Languages are you Interested in</h3>
                 <div className = "container-survey">
-                    <button onClick = {() => handleLanguageSelection('Python')}>Python</button>
-                    <button onClick = {() => handleLanguageSelection('Java')}>Java</button>
-                    <button onClick = {() => handleLanguageSelection('JavaScript')}>JavaScript</button>
-                    <button onClick = {() => handleLanguageSelection('Ruby')}>Ruby</button>
-                    <button onClick = {() => handleLanguageSelection('C')}>C</button>
+                    {['Python', 'Java','JavaScript', 'Ruby', 'C'].map((language) => (
+                        <button onClick = {() => handleLanguageSelection(language)}>{language}</button>
+                    ))}
```

Comment:
Great work refactoring this! 

You could clean this up further by defining these values in a survey-config.js file or something similar and importing and using the values directly. 

It would look something like: 
```
{survey_step_1: { heading: 'What brings you to code4all', options: ['Learning', 'Coding for Free','For Fun', 'Practicing Skills'], },
survey_step_2: {heading: 'What is your coding level', options ['Beginner', 'Intermediate','Expert'],},
...
}
```